libmamba/src/core/context.cpp

```diff
@@ -366,4 +366,23 @@ namespace mamba
         logger->dump_backtrace_no_guards();
     }
 
+    void Context::set_conda_build_local_paths(const std::vector<std::string>& vec)
+    {
+        conda_build_local_paths = vec;
+    }
+
+    std::vector<std::string> Context::get_conda_build_local_paths() const
+    {
+        if (conda_build_local_paths.empty())
+        {
+            return { prefix_params.target_prefix.string() + "/conda-bld",
+                     prefix_params.root_prefix.string() + "/conda-bld",
+                     "~/conda-bld" };
```

Comment:
Here `target_prefix` is used and therefore we can't initialize the variable in the Context creation time and therefore we need to use a getter function here.